libr/core/cmd_section.c

```diff
@@ -232,10 +264,10 @@ static int cmd_section(void *data, const char *input) {
 		}
 		break;
 	case '\0':
-		r_io_section_list (core->io, core->offset, 0);
+		print_sections (core->bin, core->offset, R_FALSE);
```

Comment:
Looking again at this, i think using print_sections instead of r_io_section_list would break all the 'S' commands ....